src/autora/doc/pipelines/main.py

```diff
@@ -1,17 +1,24 @@
 import itertools
 import logging
 from timeit import default_timer as timer
-from typing import Dict, List, Tuple
+from typing import Dict, List
 
 import torch
 import typer
 
 from autora.doc.classes.EvalResult import EvalResult
+from autora.doc.pipelines.data import load_data
 from autora.doc.pipelines.metrics import eval_bleu_meteor, eval_semscore
+from autora.doc.pipelines.train import fine_tune, get_dataset
 from autora.doc.runtime.predict_hf import Predictor
 from autora.doc.runtime.prompts import PROMPTS, PromptIds
 from autora.doc.util import get_prompts_from_file
 
+# For inference
+DEFAULT_MODEL = "meta-llama/Llama-2-7b-chat-hf"
+# For training
+DEFAULT_BASE_MODEL = "autora-doc/Llama-2-7b-chat-hf"
```

Comment:
Would it make it more clear to have the names be `DEFAULT_INFERENCE_MODEL` and `DEFAULT_TRAINING_MODEL`?